website/docs/d/tm_nsxt_manager.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_tm_nsxt_manager"
+sidebar_current: "docs-vcd-data-source-tm-nsxt-manager"
+description: |-
+  Provides a data source for available Tenant Manager NSX-T manager.
+---
+
+# vcd\_tm\_nsxt\_manager
+
+Provides a data source for available Tenant Manager NSX-T manager.
+
+Supported in provider *v3.0+*
```

Comment:
```suggestion
Supported in provider *v4.0+*
```

Reply:
Removed this at all. It is not even here before v4